pallets/pools-system/src/migrations/mod.rs

```diff
@@ -196,7 +236,94 @@ pub mod altair {
 		}
 	}
 
-	pub fn migrate_tranches<T: Config>() -> Weight
+	pub struct ScheduledUpdatePrefix;
+	impl StorageInstance for ScheduledUpdatePrefix {
+		const STORAGE_PREFIX: &'static str = "ScheduledUpdate";
+
+		fn pallet_prefix() -> &'static str {
+			"Pools"
+		}
+	}
+	pub type ScheduledUpdate<T> = StorageMap<
+		ScheduledUpdatePrefix,
+		Blake2_128Concat,
+		<T as Config>::PoolId,
+		ScheduledUpdateDetailsOf<T>,
+	>;
+
+	pub struct AccountDepositPrefix;
+	impl StorageInstance for AccountDepositPrefix {
+		const STORAGE_PREFIX: &'static str = "AccountDeposit";
+
+		fn pallet_prefix() -> &'static str {
+			"Pools"
+		}
+	}
+	pub type AccountDeposit<T> = StorageMap<
+		AccountDepositPrefix,
+		Blake2_128Concat,
+		<T as frame_system::Config>::AccountId,
+		<T as Config>::Balance,
+		ValueQuery,
+	>;
+
+	pub struct PoolDepositPrefix;
+	impl StorageInstance for PoolDepositPrefix {
+		const STORAGE_PREFIX: &'static str = "PoolDeposit";
+
+		fn pallet_prefix() -> &'static str {
+			"Pools"
+		}
+	}
+	pub type PoolDeposit<T> =
+		StorageMap<PoolDepositPrefix, Blake2_128Concat, <T as Config>::PoolId, PoolDepositOf<T>>;
+
+	fn migrate_pool_deposit<T: Config>() -> Weight {
+		let mut weight = 0u64;
```

Comment:
This weight doesn't need to be `mut`, it can just be instantiated or even used directly. for me there's no need to go and change these everywhere now since this is to be run once.